torch/linalg/__init__.py

```diff
@@ -140,6 +140,129 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+eigh = _add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
+
+This function computes the eigenvalues and eigenvectors
+of a complex Hermitian (or real symmetric) matrix :attr:`input` or batches of such matrices
+such that :math:`\text{input} = V \text{diag}(w) V^H`, where :math:`w` is the tensor of eigenvalues,
+:math:`V` is the tensor of eigenvectors and :math:`^H` is the conjugate transpose operation.
+
+Since :attr:`input` is assumed to be consisting of Hermitian matrices,
+only the lower triangular portion of these matrices is used by default (``UPLO = 'L'``)  in computations
+and the imaginary part of the diagonal will always be treated as zero.
+
+Supports real-valued and complex-valued input.
+For complex-valued batched input on CUDA the backpropagation is not supported yet.
+
+See :func:`torch.linalg.eigvalsh` for a related function that computes only eigenvalues,
+however that function is not differentiable.
+
+.. note:: The eigenvalues of real symmetric or complex Hermitian matrices are always real.
+
+.. note:: The eigenvectors of matrices are not unique, any eigenvector multiplied by a constant remains
+          to be a valid eigenvector, therefore results on CPU, CUDA and from other libraries can be different.
```

Comment:
"eigenvector, therefore results on CPU, CUDA and from other libraries can be different." -> "eigenvector. This function may compute different eigenvector representations on different device types."